GW2EIEvtcParser/EIData/ProfHelpers/Ranger/DruidHelper.cs

```diff
@@ -14,6 +14,7 @@ internal static class DruidHelper
         {
             new BuffGainCastFinder(EnterCelestialAvatar, CelestialAvatar).UsingBeforeWeaponSwap(true),
             new BuffLossCastFinder(ExitCelestialAvatar, CelestialAvatar).UsingBeforeWeaponSwap(true),
+            new BuffGainCastFinder(AncestralGraceSkill, AncestralGraceEffect),
```

Comment:
That's not an instant cast but a movement skill, It should follow a similar pattern to ComputeFlowingResolveCastEvents in WillbenderHelper.cs.